vessim/sil/redis_docker.py

```diff
@@ -1,38 +1,30 @@
-import threading
 from time import sleep
-
 import docker # type: ignore
 import redis
-import uvicorn
-from fastapi import FastAPI
 from redis import Redis
 
 
 class RedisDocker:
```

Comment:
don't we delete all of this?

Reply:
We dont need it right now. Should the whole file go for now? We have version control after all.